PowerPointLabs/PowerPointLabs/CropLab/CropLabSettingsDialog.xaml.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Windows;
+
+namespace PowerPointLabs.CropLab
+{
+    /// <summary>
+    /// Interaction logic for CropLabSettingsDialog.xaml
+    /// </summary>
+    public partial class CropLabSettingsDialog
+    {
+        public CropLabSettingsDialog()
+        {
+            SelectedAnchor = CropLabSettings.AnchorPosition;
+            InitializeComponent();
+        }
+
+        public AnchorPosition SelectedAnchor { get; set; }
+
+        private void OkButton_Click(object sender, RoutedEventArgs e)
+        {
+            CropLabSettings.AnchorPosition = SelectedAnchor;
+            Close();
+        }
+
+        private void CancelButton_Click(object sender, RoutedEventArgs e)
+        {
+            Close();
+        }
```

Comment:
CancelButton_Click can be removed after setting IsCancel="True" on the cancel button.